### ISSUE TITLE
Add `aria-expanded` and `aria-controls` attributes/functionality to va-button

### ISSUE THREAD

**Issue by jerekshoe** [department-of-veterans-affairs/vets-design-system-documentation#2752]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.
  * I found [this ticket]() that is similar, but was asked to create a separate ticket to track this bug

## What happened
When attempting to add `aria-expanded` and `aria-controls` attributes to the `button` inside the `va-button` component, the attributes are not applied. These attributes are needed for screen readers

### Chrome inspector
<img width="490" alt="Screenshot 2024-04-22 at 5 20 24 PM" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/13838621/e1d20478-502a-46dc-8fe5-0be6dc7b12a2">

### Element being inspected
<img width="500" alt="Screenshot 2024-04-22 at 5 21 47 PM" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/13838621/cf395a37-4dde-4586-9a24-2ddf382f6737">

## What I expected to happen
I expected that the button in the shadow DOM would have the `aria-expanded` and `aria-controls` attributes so that screen readers could function properly when encountering this element

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [x] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [ ] This bug is not blocking any work
- [ ] Other

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2752?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by jamigibbs** [comment]
I'm just adding a [link to the Slack conversation](https://dsva.slack.com/archives/C01DBGX4P45/p1713883175887349?thread_ts=1713821484.113299&cid=C01DBGX4P45) about this design in case it will help us prioritize the work. It sounds like the usage of button in this design context may be removed or deprecated in the next few months.
